engine/src/main/java/org/pentaho/di/job/entries/ping/JobEntryPing.java

```diff
@@ -245,6 +245,11 @@ public String getRealHostname() {
   }
 
   public String getTimeOut() {
+    if ( !isNumeric( timeout ) ) {
+      logMinimal( BaseMessages.getString( PKG, "JobPing.WarningOnlyNumericValue",
```

Comment:
Where does the the property `JobPing.WarningOnlyNumericValue` exist? I don't see it in engine/src/main/resources/org/pentaho/di/job/entries/ping/messages/messages_en_US.properties